src/components/button/button-styles.jsx

```diff
@@ -106,27 +106,88 @@ export default theme => {
       fontWeight: theme.typography.fontWeightLight,
       borderRadius: 12,
       padding: '5px 12px',
+
+      '&.iconText': {
+        paddingBottom: 5,
+        paddingTop: 4,
+      },
+      '&.icon': {
+        padding: 4,
+      },
+      '& svg': {
+        height: 11,
+        width: 11,
+      },
     },
 
     sm: {
       fontSize: 11,
       fontWeight: theme.typography.fontWeightLight,
       borderRadius: 16,
       padding: '8px 20px',
+
+      '&.iconText': {
+        paddingBottom: 7,
+        paddingTop: 7,
+      },
+      '&.icon': {
+        padding: 6,
+      },
+      '& svg': {
+        height: 13,
+        width: 13,
+      },
     },
 
     md: {
       fontSize: 14,
       fontWeight: theme.typography.fontWeightRegular,
       borderRadius: 20,
       padding: '11px 38px',
+
+      '&.iconText': {
+        paddingBottom: 10,
+        paddingTop: 10,
+      },
+      '&.icon': {
+        padding: 8,
+      },
+      '& svg': {
+        height: 16,
+        width: 16,
+      },
     },
 
     lg: {
       fontSize: 16,
       fontWeight: theme.typography.fontWeightRegular,
       borderRadius: 24,
       padding: '14px 40px',
+
+      '&.iconText': {
+        paddingBottom: 13,
+        paddingTop: 13,
+      },
+      '&.icon': {
+        padding: 10,
+      },
+      '& svg': {
+        height: 18,
+        width: 18,
+      },
+    },
+
+    '.icon': {
+      borderRadius: '50%',
+    },
+
+    innerWrapper: {
+      alignItems: 'center',
```

Comment:
What do you think about adding `justifyContent: center` as well?

Reply:
It won't do anything as the inner is just a wrapper around text+icon. That element is already centered by the padding. Is there a reason to add it?

Reply:
Ah, yeah that's true. No reason to add it then :)